### PR TITLE
[SL-157] Support Uplink Licensing

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

[SL-157]

### 🗒️ Description

Provides a way for Uplink registered plugins to be displayed in License Tab.

It will hide legacy fields when it finds the same plugin being registered through Uplink

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [x] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[SL-157]: https://stellarwp.atlassian.net/browse/SL-157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by redscar — APPROVED** [review]

**Review by redscar — APPROVED** [review]
Approved, the wpunit tests are failing due to the older version of uplink. Once we get the version of uplink correct the tests should pass without issue.

The check tests locally use `composer require stellarwp/uplink:dev-feat/fluent-form-builder --prefer-source`

File: tests/wpunit/Common/Libraries/Uplink/Controller_Test.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			dirname( __FILE__ ),
+			tribe( Tribe__Main::class )
+		);
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_setup_license_fields() {
+		// Retrieve the collection and the specific resource
```
Comment (lucatume) [inline tests/wpunit/Common/Libraries/Uplink/Controller_Test.php:45]: The current user is `0` (not logged in), should the license fields render at all?

**Review by lucatume — COMMENTED** [review]
Same question about user cap check I left on the Uplink repository.

File: src/Common/Libraries/Uplink/Controller.php [hunk 2]
```diff
@@ -0,0 +1,174 @@
+<?php
```
Comment (lucatume) [inline src/Common/Libraries/Uplink/Controller.php:1]: If all is required by the  Uplink functionality is one file, why not move it to the root?
From `./Uplink/Controller.php` to `./Uplink.php`.

**Review by lucatume — CHANGES_REQUESTED** [review]

**Review by lucatume — APPROVED** [review]

**Pull request merged**